includes/Core/Conversion_Tracking/Conversion_Event_Providers/WooCommerce.php

```diff
@@ -144,6 +151,11 @@ function ( $order_id ) use ( $input ) {
 				$order->update_meta_data( '_googlesitekit_ga_purchase_event_tracked', 1 );
 				$order->save();
 
+				$events_to_track = $this->get_event_names();
+				if ( empty( $events_to_track ) || ! isset( $events_to_track['purchase'] ) ) {
```

Comment:
Checking whether the `$events_to_track` variable is empty is useless because if it is empty then it will never contain the `purchase` key and the second condition will be truthy anyway, so we can drop it.

```suggestion
				if ( ! isset( $events_to_track['purchase'] ) ) {
```